include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -143,14 +143,14 @@ class mdspan
   )
   MDSPAN_INLINE_FUNCTION
   // TODO @proposal-bug Why is this explicit?
```

Comment:
Did we fix these TODOs in the proposal?  Had thought so.

Reply:
yeah deleted the comments.